netbox/users/views.py

```diff
@@ -106,7 +106,7 @@ def post(self, request):
             return self.redirect_to_next(request, logger)
 
         else:
-            logger.debug("Login form validation failed")
+            logger.debug(f"Login form validation failed for username: {form['username'].value()}")
```

Comment:
We should be able to get the username from the cleaned data:
```suggestion
            logger.debug(f"Login form validation failed for username: {form.cleaned_data['username']}")
```

Reply:
Not sure on this one - it is in a case where the form is not valid, from the docs "If your data does not validate, the cleaned_data dictionary contains only the valid fields:" so it is potential that the username is not in the cleaned_data in this case.